kratos/python_scripts/KratosUnittest.py

```diff
@@ -229,7 +230,14 @@ def runTests(tests):
     # parse command line options
     parser = argparse.ArgumentParser()
 
-    parser.add_argument('-l', '--level', default='all', choices=['all', 'nightly', 'small', 'validation'])
+    # Depending of serial/MPI the level can be different, so we have to parse it
+    if KM.IsDistributedRun():
+        try: # Try to parse the command line with mpi_ in first place
+            parser.add_argument('-l', '--level', default='mpi_small', choices=['mpi_all', 'mpi_nightly', 'mpi_small', 'mpi_validation'])
+        except: # If mpi_ is not found we will parse with "serial" input, it will be parsed later
+            parser.add_argument('-l', '--level', default='small', choices=['all', 'nightly', 'small', 'validation'])
```

Comment:
The `except` clause is never reached here because the eventual exception would be raised on `parser.parse_args()`. I would propose something like the following, which is less elegant but closer to the actual operative.
```suggestion
        parser.add_argument('-l', '--level', default='mpi_small', 
            choices=['mpi_all', 'mpi_nightly', 'mpi_small', 'mpi_validation', 'all', 'nightly', 'small', 'validation'],
            help="For distributed runs, options that don't start with mpi_ will be replaced by the corresponding mpi version")
```